mkdocs-requirements.txt

```diff
@@ -0,0 +1,4 @@
+mkdocs
```

Comment:
Do we want to pin versions here?

Reply:
Yes please 👍🏼 I'm assuming there will be other missing bits needed to make the whole thing work correctly. WDYT about me adding these `make` targets to this PR along with a blurb in `docs/CONTRIBUTING.md`?

- `make docs-deps`
- `make docs`
- `make serve-docs`

Reply:
Makes sense to me!